### ISSUE TITLE
new changes

### ISSUE THREAD

**Issue by xzrvr** [sbeeo/SAM-Auto#3]
hi 
u should make a option to check if the acheivement is already unlocked and only obtain the unobtained acheivments :]

**Comment by Czekd0ut** [comment]
I agree that this should be a thing. Tired of going through games that have already had all their achievements unlocked. It sucks especially when you have a larger library and have to go through all of them each time I run the program.
